randovania/games/am2r/json_data/Distribution Center.txt

```diff
@@ -963,19 +963,26 @@ Extra - minimap_data: [{'x': 70, 'y': 44}, {'x': 70, 'y': 45}, {'x': 70, 'y': 46
   * Extra - instance_id: 134710
   > Door to Bullet Hell Room
       Trivial
+  > EMP Ball
+      Any of the following:
+          Morph Ball and Morph Glide (Advanced)
```

Comment:
Is this easier with HJ?

Reply:
No, in fact I found the opposite to be true since you reach the ceiling too quickly. Non-HJ allows you to carry the vertical momentum for longer.